README.md

```diff
@@ -61,3 +61,40 @@ As mentioned above, package specifications are versioned. Versions follow the [s
 ## Version Compatibility between Packages and Specifications
 
 A package specifying its `format_version` as `x.y.z` must be valid against specifications in the semantic version range `[x.y.z, X.0.0)`, where `X = x + 1`.
+
+
+# What is an Elastic Package?
+
+An Elastic Package is a collection of assets for the Elastic Stack. In addition, it contains manifest files which contain additional information about the package. The exact content and structure of a package are described by the package spec. In the following is a high level overview of a package.
```

Comment:
Maybe add something about how packages also contain information about configuring them via the Fleet UI?

Reply:
Added